### PR TITLE
Document System.Security.Cryptography.Pkcs.SignedCms

### PR DESCRIPTION
**Pull request by carlossanlop**
Proposal: https://github.com/dotnet/corefx/issues/25449
PR that added the code: https://github.com/dotnet/corefx/pull/29408

### CODE REVIEW

File: xml/System.Security.Cryptography.Pkcs/SignedCms.xml [hunk 1]
```diff
@@ -926,8 +927,8 @@ The following permissions are required to access the signature key:
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="certificate">The certificate to remove from the collection.</param>
+        <summary>Removes the specified certificate from the collection of certificates for the encoded CMS/PKCS #7 message, if it exists.</summary>
         <remarks>To be added.</remarks>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Pkcs/SignedCms.xml]: The source shows that the method throws a CryptographicException if the certificate is not found: 
image

File: xml/System.Security.Cryptography.Pkcs/SignedCms.xml [hunk 2]
```diff
@@ -926,8 +927,8 @@ The following permissions are required to access the signature key:
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="certificate">The certificate to remove from the collection.</param>
+        <summary>Removes the specified certificate from the collection of certificates for the encoded CMS/PKCS #7 message, if it exists.</summary>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Pkcs/SignedCms.xml]:
```suggestion
        <summary>Removes the specified certificate from the collection of certificates for the encoded CMS/PKCS #7 message.</summary>
```
The "if it exists" implies that if it doesn't, nothing happens.

Comment (carlossanlop) [reply xml/System.Security.Cryptography.Pkcs/SignedCms.xml]: My bad. I did not see the last line. I'll update the description.

**Comment by carlossanlop** [comment]
Suggestions addresed, @rpetrusha, thank you for noticing.
@krwq can you please make sure the descriptions I added are accurate?

**Comment by rpetrusha** [comment]
Is this ready to merge, @carlossanlop?

**Comment by rpetrusha** [comment]
Thanks, @krwq. I'll merge now, @carlossanlop.

**Pull request merged**